### ISSUE TITLE
Slots inside a scoped slot won’t be generated

### ISSUE THREAD

**Issue by runarberg** [HcySunYang/vuese#119]
**Describe the bug**
Slots that are children of a component that uses scoped slots (i.e. `v-slot`) won’t be generated.

**To Reproduce**
Create a component that has a slot inside a scoped slot from another component.
```vue
<template>
  <FooBar v-slot="{ foo }">
    <!-- Some header -->
    <slot name="header">Title</slot>
  </FooBar>
</template>
```

**Expected behavior**
There should be a section in the generated docs about the `<slot name="header">`.

**Environmental information:**
```
Environment Info:

  System:
    OS: Linux 5.3 Ubuntu 19.10 (Eoan Ermine)
    CPU: (4) x64 Intel(R) Core(TM) i7-7600U CPU @ 2.80GHz
  Binaries:
    Node: 12.12.0 - ~/.nvm/versions/node/v12.12.0/bin/node
    Yarn: 1.19.1 - ~/work/stemson/ui/node_modules/.bin/yarn
    npm: 6.11.3 - ~/.nvm/versions/node/v12.12.0/bin/npm
  Browsers:
    Firefox: 70.0.1
```

**Comment by IWANABETHATGUY** [comment]
Duplicate of #72

**Comment by IWANABETHATGUY** [comment]
will be fixed by #161
